PriscillaSam/Stackoverflow-Lite
# Implement LFA's feedback on code base

#### What does this PR do?
Make changes to code base based on LFA observations.

#### Description of task to be completed
- Rename api endpoint to vote an answer from `/api/v1/answers/:id` to `/api/v1/answers/:id/votes`.
- Make changes to variable names.
- Update tests based on changes made.

#### How can this manually tested?
* Clone the repo using `git clone https://github.com/PriscillaSam/Stackoverflow-Lite.git`.
* run `git checkout bg-implement-feedback`
* run `npm install` and `npm start`.
* Open up Postman and test **POST** `localhost:3000/api/v1/answers/:id/votes`.

Request - **POST**  `localhost:3000/api/v1/answers/6/votes`
```gherkin
{
   vote: 0
}
```

Response should be: 
```gherkin
{
    "status": "success",
    "message": "you have downvoted this answer",
    "answer": {
        "id": 6,
        "answer": "I really am not sure but I feel like God wants it so.",
        "questionid": 3,
        "userid": 4,
        "upvotes": "2",
        "downvotes": "1"
    }
}
```

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
#160022206
